tasks/e2e.sh

```diff
@@ -28,6 +28,30 @@ function handle_exit {
   exit
 }
 
+function build_check {
+  local test_snap_path="$1"
+  if [ -z "$1" ]
```

Comment:
This is just an end-to-end test that *we* run for this tool, the developer won’t see it.
So relying on jest existing is fine here.